virtwho/register.py

```diff
@@ -261,8 +263,195 @@ def facts_remove(self):
 
 
 class RHSMAPI:
-    def __init__(self):
-        pass
+
+    def __init__(self, host, username, password, port=22):
```

Comment:
About API interface, it's more better if we can use python request module to get and post data to instead of curl.  we have similar implement as kubevirt_request in the old provision.py.

Reply:
Changed to use the requests module, which is more recommended by compared.